libs/editor/egui_editor/src/offset_types.rs

```diff
@@ -438,6 +439,10 @@ where
         *max(&self.0, &self.1)
     }
 
+    fn len(&self) -> <T as Sub>::Output {
```

Comment:
fun rust syntax for "a function of `&self` which returns whatever would be returned by subtracting one instance of the inner element type from another" (e.g. `RelCharOffset` when trait is implemented for `(DocCharOffset, DocCharOffset)`)